cpp/src/parquet/reader_test.cc

```diff
@@ -778,6 +782,43 @@ TEST_F(TestCheckDataPageCrc, CorruptDict) {
   }
 }
 
+class TestGzipMembersRead : public ::testing::Test {
```

Comment:
This class isn't actually used anywhere. `TEST` defines a test function, you would use `TEST_F` for a test method. Can you remove dead code?

(as a general guideline: please re-read yourself when submitting a PR)